requirements/run.in

```diff
@@ -1,2 +1,3 @@
 apscheduler
 requests
+glom
```

Comment:
The dependency of amlight/sdntrace_cp shouldn't be listed here?

Reply:
No, NApps are not installed like regular python packages.